.github/workflows/build-and-run-tests-windows.yml

```diff
@@ -18,9 +18,9 @@ jobs:
       uses: actions/checkout@v3
 
     - name: Setup Python
-      uses: actions/setup-python@v2
+      uses: actions/setup-python@v4
       with:
-        python-version: '3.x'
+        python-version: '3.10'
```

Comment:
Why 3.10 and not 3.x which is 3.11 at the moment?

Reply:
... because I did not understand the [x-ranges](https://github.com/npm/node-semver#x-ranges-12x-1x-12-) stuff. Feel free to change back to 3.x.